Add heroku deployment dependencies (#11)
EBIvariation/trait-curation: docs/heroku-deployment.md

> 
> Step by step guide on how to deploy the app in Heroku and enable automatic pull request deployment as well.

Reviewer: ```suggestion
# Setting up deployment in Heroku and automated pull request deployment
```

Reviewer: Removing blank line from the start of the file and converting the first sentence into a header

The instructions assume that a Heroku account has been set up.

> 
> 

Reviewer: ```suggestion

```
Extra blank line
